datastructure/과제는 끝나지 않아!.kt

```diff
@@ -0,0 +1,38 @@
+// https://www.acmicpc.net/problem/17952
+
+import java.util.*
+
+var score = 0
+val stack = Stack<Homework>()
+
+fun main() {
+    val N = readLine()!!.toInt()
```

Comment:
- 이렇게 해도 좋을 것 같아요~! non null을 명시하지 않아도 되니!
```suggestion
    val N = readln().toInt()
```

Reply:
앗,, 맞네용 ㅎㅎ readln()이 있었죠~! 감사합니당 :)